avm/res/api-management/service/api/main.bicep

```diff
@@ -63,8 +66,11 @@ param format string = 'openapi'
 @description('Optional. Indicates if API revision is current API revision.')
 param isCurrent bool = true
 
+@description('Optional. The name of the API management service logger.')
+param loggerName string = ''
+
 @description('Required. Relative URL uniquely identifying this API and all of its resource paths within the API Management service instance. It is appended to the API endpoint base URL specified during the service instance creation to form a public URL for this API.')
-param path string
+param apiPath string
```

Comment:
Should be api as per spec [Inputs - Parameter/Variable Naming](https://azure.github.io/Azure-Verified-Modules/specs/shared/#id-rmfr6---category-inputs---parametervariable-naming)

Background: As it's the `path` parameter of the `api` resource type it should be intuitive that it's the api's path. Following this logic it would be redundant to call it `apiPath`